packages/providence/README.md

```diff
@@ -0,0 +1,215 @@
+# Providence
+
+Providence is the 'All Seeing Eye' that generates usage statistics by analyzing code.
+It measures the effectivity and popularity of your software.
+In just a few clicks you can measure the impact for (breaking) changes; making
+your release process more stable and predictable.
+
+## Setup
+
+### Install providence
+
+```sh
+npm i --save-dev providence
+```
+
+### Add a providence script to package.json
+
+```js
+...
+"scripts": {
+    ...
+    "providence": "providence analyze match-imports -r 'node_modules/lion-based-ui'",
```

Comment:
The reference project would usually be node_modules/@lion/package right? lion-based-ui is the target usually right where you run providence in its root?

I wonder if in general we should have a command/flag that supports multi-packages, `-r 'node_modules/@lion' --mono-repo` or something, I dunno. Maybe not in the scope of this story though.

Reply:
Edit: Maybe I'm confused about what the perspective is in these docs.

Reply:
No, you're right :) will fix it